### PR TITLE
add registration form

### PR DESCRIPTION
**Pull request by vilhota30**

### CODE REVIEW

**Review by tymnik — COMMENTED** [review]
Page should be named like RegistrationPage and it is usual to make a default export.
Files with css should be named .module.css and styles should be applied to elements with className={css.divFormRegister}
Labels for inputs are absent.

File: src/pages/Registration.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+.register-btn {
+    padding: 4px;
+    border-radius: 6px;
+}
+
+.register-notification {
+    color: red;
+    font-size: 14px;
+}
```
Comment (StanislavKharkiv) [inline src/pages/Registration.css]: doesn't matter how to call class camelCase or kebab-case. But discuss this question with team and choose how you will do it and leave only one name style in all css files

File: src/pages/Registration.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+export const FormRegistration = () => {
+    const schema = yup.object().shape({
+        username: yup.string().min(4).required("username is Required!"),
+        email: yup.string().email().required("email is Required!"),
+        password: yup.string().min(4).max(20).required(),
+        repeat_password: yup.string().oneOf([yup.ref("password"), undefined], "passwords don't match!").required(),
+    });
+
```
Comment (StanislavKharkiv) [inline src/pages/Registration.tsx]: please put schema out of component, if component big(more than 100 stings) better move schema in separate file

File: src/pages/Registration.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        //     });
+        //     if (response.ok) {
+        //         console.log('Form submitted successfully!');
+        //         reset();
+        //     } else {
+        //         console.error('Error submitting form:', response.statusText);
+        //     }
+        // } catch (error) {
+        //     console.error('Error submitting form:', error.message);
+        // }
```
Comment (StanislavKharkiv) [inline src/pages/Registration.tsx]: use axios for send data

Comment (vilhota30) [reply src/pages/Registration.tsx]: corrected

File: src/pages/Registration.tsx (hunk 2 above)
Comment (vilhota30) [reply src/pages/Registration.tsx]: corrected

File: src/pages/Registration.css (hunk 1 above)
Comment (vilhota30) [reply src/pages/Registration.css]: corrected

**Review by StanislavKharkiv — APPROVED** [review]

**Pull request merged**